.github/workflows/release.yml

```diff
@@ -242,3 +242,56 @@ jobs:
 
     - name: Publish package distributions to PyPi
       uses: pypa/gh-action-pypi-publish@release/v1
+
+  wasm-package:
+    strategy:
+      fail-fast: false
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Create package
+      run: npm pack f3d-app/f3d#${{github.event.inputs.f3d_version}}
+
+    - name: Rename package
+      run: for file in f3d-*.tgz ; do mv $file ${file//f3d-/f3d-web-} ; done
+
+    - name: Publish npm package
+      uses: ncipollo/release-action@v1
+      with:
+        owner: f3d-app
+        repo: f3d
+        token: ${{ secrets.SECRET_TOKEN }}
+        prerelease: true
+        allowUpdates: true
+        updateOnlyUnreleased: true
+        replacesArtifacts: true
+        omitBodyDuringUpdate: true
+        omitNameDuringUpdate: true
+        tag: ${{github.event.inputs.f3d_version}}
+        artifacts: './f3d-web-*.tgz'
+
+  npm-publish:
+    needs: wasm-package
+    strategy:
+      fail-fast: false
+
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+
+    steps:
+    - uses: actions/download-artifact@v4
+      with:
+        pattern: f3d-web-*
+
+    - run: npm publish --access public --tag next f3d-web-*.tgz
```

Comment:
what does this `next` tag do ? why do we need it ?

Reply:
ha, it for uploading RCs :)

Reply:
you could use github action logic to only add the `--tag next` if contains RC and do both steps in a single one